src/neuroglancer/ui/default_viewer.ts

```diff
@@ -14,23 +14,11 @@
  * limitations under the License.
  */
 
-require('./default_viewer.css');
+import './default_viewer.css';
```

Comment:
I see that the point of this is to allow the code here to be used without the CSS import, since the CSS code is pretty specific to the default viewer and wouldn't really be applicable when embedding neuroglancer in another application.

However, some of the other stuff here is perhaps also questionable --- like disableContextMenu() and disableWheel(), since those apply to the entire document, and picking the neuroglancer-container element by ID is also a bit unfortunate for an embedding use case.

In any case, if you really do want everything here except the CSS, a simpler option would be to just move the css import to main.ts and main_python.ts.

Reply:
Those are all valid points. I have moved the disableContextMenu() and disableWheel() methods back into the makeDefaultViewer function. I have also added a target option to makeMinimalViewer() which will allow the calling code to specify a target dom element for rendering.